### PR TITLE
Improvement/gh 2335 create testrail objects

### PR DESCRIPTION
**Pull request by TeddyAndrieux**
**Component**:

'build', 'tests'

**Context**: 

To upload test result to testrail we first need to create testrail objects

**Summary**:

Create a YAML file describing all the TestRail objects we want to create and during post-merge run a script to create all these testrail objects

---

Fixes: #2335

### CODE REVIEW

File: eve/main.yml [hunk 1]
```diff
@@ -417,6 +433,44 @@ stages:
...
+        docker-basic: eve/workers/docker-centos7
+      vars:
+        name: "metalk8s-test-tools-worker"
+    steps:
+      - Git: *git_pull
+      - ShellCommand: &prepare_test_tools
+          name: Install test tools requirements
+          command: >
+            git clone git@bitbucket.org:scality/test_tools.git
+            --branch "improvement/add_new_testrail_lib" &&
```
Comment (TeddyAndrieux) [inline eve/main.yml]: FYI: This line will be removed once merged in master

**Review by alexandre-allard — APPROVED** [review]

File: eve/main.yml [hunk 2]
```diff
@@ -363,6 +372,7 @@ stages:
             - lint
           haltOnFailure: true
       - SetPropertyFromCommand: *set_version_prop
+      - SetPropertyFromCommand: *set_short_version_prop
```
Comment (alexandre-allard) [inline eve/main.yml:375]: Does not seem to be used.

Comment (TeddyAndrieux) [reply eve/main.yml:375]: No indeed in premerge we not use it, but IMO having a property for short_version it's good anyway

**Pull request merged**